Vector/Model/RoomList/RecentsDataSource.m

```diff
@@ -20,6 +20,21 @@
 
 #import "EventFormatter.h"
 
+@interface RecentsDataSource()
+{
+    NSMutableArray* favoritesCells;
```

Comment:
I suggest renaming 'favoritesCells' with 'favoritesCellDataArray'
(idem for conversation and low priority)